### ISSUE TITLE
ImportError: cannot import name 'soft_unicode' from 'markupsafe'

### ISSUE THREAD

**Issue by Gabbinetto** [berinhard/pyp5js#212]
I installed the library, it went fine, but as soon as I try to run a command such as `pyp5js new sketch` it gives a long error ending in `ImportError: cannot import name 'soft_unicode' from 'markupsafe'`.
Here it is the full error:
```
Traceback (most recent call last):
  File "C:\Users\gabri\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\gabri\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\gabri\AppData\Local\Programs\Python\Python310\Scripts\pyp5js.exe\__main__.py", line 4, in <module>
  File "C:\Users\gabri\AppData\Local\Programs\Python\Python310\lib\site-packages\pyp5js\cli.py", line 8, in <module>
    from pyp5js import commands
  File "C:\Users\gabri\AppData\Local\Programs\Python\Python310\lib\site-packages\pyp5js\commands.py", line 5, in <module>
    from jinja2 import Environment, FileSystemLoader
  File "C:\Users\gabri\AppData\Local\Programs\Python\Python310\lib\site-packages\jinja2\__init__.py", line 12, in <module>
    from .environment import Environment
  File "C:\Users\gabri\AppData\Local\Programs\Python\Python310\lib\site-packages\jinja2\environment.py", line 25, in <module>
    from .defaults import BLOCK_END_STRING
  File "C:\Users\gabri\AppData\Local\Programs\Python\Python310\lib\site-packages\jinja2\defaults.py", line 3, in <module>
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
  File "C:\Users\gabri\AppData\Local\Programs\Python\Python310\lib\site-packages\jinja2\filters.py", line 13, in <module>
    from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (C:\Users\gabri\AppData\Local\Programs\Python\Python310\lib\site-packages\markupsafe\__init__.py)
```
I don't know what this all means.

**Comment by kishiyu** [comment]
I have same problem.
I use windows 11, Python 3.8.2.

**Comment by ahdistefano** [comment]
Same here with Python 3.9.7. Looks like there might be a conflict with the dependencies versions.

**Comment by berinhard** [comment]
Hi everyone. Thanks for trying pyp5js and reporting this issue. The fix maybe took longer than you've expected, but please consider that `pyp5js` is my personal project that I can only find time to work on when I have some spare time. I moved from South America to Europe this mid-year and only by now I manage to settle myself and find some time to get back to open source activity.

I've released a new [0.7.2 version](https://pypi.org/project/pyp5js/0.7.2/) with dependencies updates and a more explicit support to Python 3.10 with the Github actions. Can you maybe try to run `$ pip install -I pyp5js==0.7.2` and test pyp5js again with a newer version?

Thanks a lot!
